tests/testthat/test-capture-requests.R

```diff
@@ -42,6 +43,7 @@ test_that("We can record a series of requests (a few ways)", {
     "httpbin.org/image/webp.R-FILE", # The `write_disk` location
     "httpbin.org/put-PUT.json", # Not a GET, but returns 200
     "httpbin.org/response-headers-ac4928.json",
+    "httpbin.org/status/200.txt", # 200 response, so .txt
```

Comment:
Let's clarify what this is

```suggestion
    "httpbin.org/status/200.txt", # empty 200 response "text/plain", so .txt
```